### PR TITLE
Svelte: use Inter font for --font-family-base

### PR DESCRIPTION
**Pull request by camdencheek**
This adds `Inter` as the preferred for `--font-family-base` in the Svelte webapp. Previously, we were using system fonts for our base font, which meant our web app looked different depending on the OS+browser.

Instead of depending on the Google Fonts CDN, this uses Fontsource, which let us easily self-host Google Fonts (and other OSS fonts) just by importing an NPM package. No committing font files to git, and no dependency or connection to a 3rd-party server. The font itself weighs ~40KB for the full, variable-weight font. 

Contributes to SRCH-445

## Test plan

`pnpm dev` and `sg start`, inspect element to ensure that we were rendering the new font, and check the network tab to ensure we weren't hitting the CDN and that the font wasn't super large. The

### CODE REVIEW

File: doc/dependency_decisions.yml [hunk 1]
```diff
@@ -615,3 +615,9 @@
     :why:
     :versions: []
     :when: 2024-06-05 15:32:44.440111000 Z
+- - :permit
+  - OFL-1.1
+  - :who:
+    :why: Safe open font license
+    :versions: []
+    :when: 2024-06-12 01:39:30.854523000 Z
```
Comment (camdencheek) [inline doc/dependency_decisions.yml:623]: Added the OFL license to the list of allowed licenses. This is not a risky license, we are not modifying the font, and we are distributing the license with our list of OSS dependencies.

File: client/web-sveltekit/src/lib/styles/fonts.scss
Comment (fkling) [inline client/web-sveltekit/src/lib/styles/fonts.scss:1]: Are we importing this anywhere?

**Review by fkling — APPROVED** [review]
I'm not super familiar with fonts, but these changes seem pretty straightforward.

File: client/web-sveltekit/src/lib/styles/fonts.scss
Comment (camdencheek) [reply client/web-sveltekit/src/lib/styles/fonts.scss:1]: Oh whoops. Leftover from a different attempt.

**Pull request merged**